### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by karot9100** [TormStorm/jagex-launcher-linux#66]
Since your last change to the yaml file, i can't install this anymore on my arch based distro. No permission to create /wine

**Comment by igreisz** [comment]
Also running into the same issue. 

Using flatpak Lutris on Fedora. 

Is it because it's missing the executable permissions?

![image](https://github.com/TormStorm/jagex-launcher-linux/assets/37279489/62fc5961-ce7e-4779-a819-c1395e660596)

**Comment by igreisz** [comment]
Also unable to install with this new version of the .yml. Running Fedora with Lutris from Flatpak. Not sure what the root of the issue is - though the launcher timing out seems to be my main issue.

**Comment by karot9100** [comment]
I've downloaden an older version of the yml file and i managed to get it installed. Still did not run but i had to disable the nvidia offprime rendering thing in lutris so it could start. I'm running intel/nvidia rtx on endeavor os. Fresh install.

**Comment by ithier** [comment]
@karot9100 which commit specifically worked for you ?

**Comment by karot9100** [comment]
> @karot9100 which commit specifically worked for you ?

One from about a month ago. The one before lutris ge 8-10 requirement got removed.

**Comment by ithier** [comment]
@karot9100 Cheers.

Can confirm. The latest commit that works is: 8c9c6bff84517dc989909c0480ee72ba885891d9

Every other commit / merge after that has some kind of error.

**Comment by TormStorm** [comment]
@ithier Are you using Lutris from Flathub or from your distro? What error message are you getting?

**Comment by ithier** [comment]
@TormStorm Ah right, I have Lutris installed from the distro.

The error from the latest commit:

```
mkdir: cannot create directory ‘/wine’: Permission denied
Monitored process exited.
Initial process has exited (return code: 256)
All processes have quit
Exit with return code 256
```

**Comment by karot9100** [comment]
This was the same issue i was having. Im using another distro by now so i cannot check which lutris version i had. It was the one from pop os inside the pop shop.

The one im using now is arch based and its another version of lutris and it works alot better (with installing with an older yml file.)

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by TormStorm** [comment]
> @TormStorm Ah right, I have Lutris installed from the distro.
> 
> The error from the latest commit:
> 
> ```
> mkdir: cannot create directory ‘/wine’: Permission denied
> Monitored process exited.
> Initial process has exited (return code: 256)
> All processes have quit
> Exit with return code 256
> ```

Looks like it's trying to create the Wine folder in your root directory which it's not suppose to do. If you install Lutris from Flathub like recommended it should be resolved. See https://github.com/TormStorm/jagex-launcher-linux/issues/64#issuecomment-1673099018

**Comment by KafoKajKodo** [comment]
I have received this error using a shell other than bash, it was fixed on switching to bash. Unsure if this applies to you.

**Comment by TormStorm** [comment]
> I have received this error using a shell other than bash, it was fixed on switching to bash. Unsure if this applies to you.

It's a problem with Lutris from your distribution. Switch to the Flatpak and it should resolve the issue.

**Comment by karot9100** [comment]
Thanks for the reply! I will check it out once i have to reinstall it!

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by TormStorm** [comment]
Addressed with #73
